web/packages/plugins/apps/analysis-platform/src/modules/Asset/components/CreateAssetForm.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react'
+import { Button, TextField } from '@equinor/eds-core-react'
+import { INPUT_FIELD_WIDTH, TLocation } from '@dmt/common'
+import { TAsset } from '../../../Types'
+import { EBlueprints } from '../../../Enums'
+import { FormWrapper } from '../../../components/Design/Styled'
+
+type Errors = {
+  [key: string]: any
+}
```

Comment:
How is this different from the "any" type?

Reply:
Not my code so limited insight, but I'd guess it differs in that it must be a `key:val` dict rather than any object (which also includes lists, etc.)?